packages/@sfx/sayt-driver-plugin/src/sayt-driver-plugin.ts

```diff
@@ -75,15 +75,18 @@ export default class SaytDriverPlugin implements Plugin {
   }
 
   /**
-   * Dispatches an event with the response from the sayt data plugin.
+   * Dispatches an event with the response from the sayt data plugin and the origin searchbox id.
    * Callback for the Sayt data request event listener.
    *
    * @param event Event that contains the Sayt API request payload.
    */
   fetchSaytData(event: CustomEvent): void {
+    const searchbox = { searchbox: event.detail.searchbox };
     this.sendSaytApiRequest(event.detail)
```

Comment:
This will now send the searchbox id to the Sayt API. We should try and prevent sending any pieces of information that are not expected. I would suggest changes to deconstruct the original `event.detail` in this function and only pass the required pieces to `sendSaytAPIRequest()`.

Reply:
@geoffcoutts I don't think Im sending the searchbox id to the API, I thought what I was doing was getting the response from the api, then() adding the searchbox to the response and sending searchbox with response in the event on line 89

Reply:
The ID is part of `event.detail` which is getting passed to `sendSaytAPIRequest()`. This deconstructs the event into `query` and `config`, which will now contain the ID, and sends both pieces to the API.